modules/xml/src/test/java/org/mule/transformers/xml/XmlToXMLStreamReaderTestCase.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.transformers.xml;
+
+import org.mule.api.MuleEvent;
+import org.mule.api.transformer.DataType;
+import org.mule.api.transformer.Transformer;
+import org.mule.api.transport.OutputHandler;
+import org.mule.module.xml.transformer.XmlToDomDocument;
+import org.mule.module.xml.transformer.XmlToXMLStreamReader;
+import org.mule.module.xml.util.XMLUtils;
+import org.mule.transformer.AbstractTransformerTestCase;
+
+import java.io.IOException;
+import java.io.OutputStream;
+
+import javax.xml.stream.XMLInputFactory;
+import javax.xml.stream.XMLStreamException;
+import javax.xml.stream.XMLStreamReader;
+
+import org.custommonkey.xmlunit.XMLUnit;
+import org.junit.Test;
+import org.w3c.dom.Document;
+
+public class XmlToXMLStreamReaderTestCase extends AbstractTransformerTestCase
```

Comment:
The name of this test doesn't represent what you're doing. This test can only be applied to test the output handler case. Reflect that on the name